angular/projects/admin/src/app/admin/document/document-list/document-list-datasource.ts

```diff
@@ -13,48 +13,42 @@ export class DocumentListDataSource extends DataSource<Document> {
     private readonly paginator: MatPaginator,
     private readonly sort: MatSort,
     private readonly status: string,
+    private readonly nextPage: Boolean,
```

Comment:
Use basic type `boolean` (https://www.typescriptlang.org/docs/handbook/declaration-files/do-s-and-don-ts.html)

Reply:
Resolved on this commit https://github.com/oddbit/tanam/pull/220/commits/11cbc5687b780dba355845f7402967bed916a2d6